docs/source/daemon/config.rst

```diff
@@ -16,15 +16,53 @@ The file extension determines the type of configuration interface to use -
     $ python3 -m cobald.daemon /etc/cobald/config.py
 
 The YAML Interface
-------------------
+==================
 
 The top level of a YAML configuration file is a mapping with two sections:
 the ``pipeline`` section setting up a pool control pipeline,
 and the ``logging`` section setting up the logging facilities.
-The ``logging`` section is optional and follows the standard `configuration dictionary schema`_.
+The ``logging`` section is optional and follows the standard
+`configuration dictionary schema`_.
 
-The ``pipeline`` section contains a sequence of :py:class:`~cobald.interface.Controller`,
-:py:class:`~cobald.interface.Decorator` and :py:class:`~cobald.interface.Pool`\ s.
+The ``pipeline`` section must contain a sequence of
+:py:class:`~cobald.interface.Controller`\ s,
+:py:class:`~cobald.interface.Decorator`\ s
+and :py:class:`~cobald.interface.Pool`\ s.
```

Comment:
Do I really need a combination of all three of them?

```suggestion
and/or :py:class:`~cobald.interface.Pool`\ s.
```

Reply:
At least a Controller and Pool, but Decorators are optional.